R/get_marginaleffects.R

```diff
@@ -1,6 +1,4 @@
-#' @rdname get_marginalmeans
-#'
-#' @inheritParams get_emmeans
+#' @rdname get_emmeans
```

Comment:
It doesn't like that 

`✖ In topic 'get_marginalmeans.Rd': Skipping; no name and/or title.`

what did I forget